packages/babel-generator/src/node/whitespace.ts

```diff
@@ -200,16 +200,19 @@ export const nodes: {
  * Test if Property needs whitespace.
  */
 
-nodes.ObjectProperty = nodes.ObjectTypeProperty = nodes.ObjectMethod = function (
-  node: t.ObjectProperty | t.ObjectTypeProperty | t.ObjectMethod,
-  parent: any,
-): WhitespaceObject | undefined | null {
-  if (parent.properties[0] === node) {
-    return {
-      before: true,
+nodes.ObjectProperty =
+  nodes.ObjectTypeProperty =
+  nodes.ObjectMethod =
+    function (
```

Comment:
Prev was better

Reply:
The first line is longer than 80. Prettier's not splitting that line was a bug. What can an alternative look like in this case?

Reply:
Oh then this is ok.

Or maybe
```js
nodes.ObjectProperty = nodes.ObjectTypeProperty = nodes.ObjectMethod =
  function (
```

since I feel like it gives the same importance to all the member expressions, but it's not a strong preference.